CHANGELOG.md

```diff
@@ -6,6 +6,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+- more [Task Enum](./README.md#task-enum) tasks
+- [accelerator](./README#accelerators) options in [Runtime Object](./README#runtime-object)
+- [Model Output Object](./README.md#model-output-object)
+- batch_size and hardware summary
+- [`disk_size`, `memory_size`](./README#architecture-object)
+- [`hardware_summary`, `accelerator`, `accelerator_constrained`](./README#runtime-object) to specify hardware requirements for inference
+- Use common metadata [Asset Object](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#asset-object) to refer to model asset and source code.
```

Comment:
To revise according to other comment about model asset as actual STAC Assets under `assets`.